CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+## Questions
+
+If you have questions about implementation details, help or support, then please use our dedicated community forum at (DISCORD)[https://discord.com/channels/456315379489308693/456317644539428875].
```

Comment:
The Discord link is not the dedicated forum; please use `https://discord.com/channels/809809541385682964/809827655770832916` instead.